setup.py

```diff
@@ -5,22 +5,22 @@
 
 setup(
     name="graviton",
-    version="0.4.1",
+    version="0.5.0",
     url="https://github.com/algorand/graviton",
     description="verify your TEAL program by experiment and observation",
     long_description=long_description,
     author="Algorand",
     author_email="pypiservice@algorand.com",
     python_requires=">=3.9",
-    install_requires=["py-algorand-sdk==1.15.0", "tabulate==0.8.9"],
+    install_requires=["py-algorand-sdk==1.20.0", "tabulate==0.9.0"],
```

Comment:
@tzaffi Considered post-approval:  At least for py-algorand-sdk, can `install_requires` dependencies be loosened to > X and < Y versions?

* It's unclear to me that the highlighted versions are _strictly_ required for a working installation.  I assume a py-algorand-sdk version providing the new dryrun fields is _strictly_ required (https://github.com/algorand/py-algorand-sdk/releases/tag/v1.14.0).
* Reference:  https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#id5.

Reply:
Thanks for research into the sensible dependencies here. I was thinking a bit ahead to the case of testing source maps where I wanted a [@barnjamin 's bugfix](https://github.com/algorand/py-algorand-sdk/releases/tag/v1.16.1) to be available, and so putting in the latest py-algorand-sdk dependency was the easiest way to achieve that. But even here, I need only insist on `>=1.16.1`. So I'll relax the dependency to `1.16.1` as the lower bound, and re-test CI. I'll also relax the restriction in the companion PyTeal PR.

I'm open to further discussion.